examples/java/src/test/java/org/apache/beam/examples/cookbook/TriggerExampleTest.java

```diff
@@ -104,7 +104,7 @@ public void testExtractTotalFlow() throws Exception {
     List<KV<String, Integer>> results = extractFlowInfow.processBundle(INPUT);
     Assert.assertEquals(results.size(), 1);
     Assert.assertEquals(results.get(0).getKey(), "94");
-    Assert.assertEquals(results.get(0).getValue(), new Integer(29));
+    Assert.assertEquals(results.get(0).getValue(), Integer.valueOf(29));
```

Comment:
It's not related to UTF-8 here, right ? Maybe having two different commits would be good.

Reply:
OK I'll remove those changes.

Reply:
Great, thanks. I would like to avoid unrelated changes in the same commit.

Reply:
OK I removed those changes from the PR.